### ISSUE TITLE
Autoload error -Vendor autoload is not found. Please run 'composer install' under application root directory.

### ISSUE THREAD

**Issue by JackWormUK** [magento/magento2-kubernetes-devbox#72]
Hi.

I have pretty much managed to get there with this on both Windows (with a small mod) and on Mac

However both are giving the Autoload error and both hang with bash m-composer install.  Is there a way to install composer or is there something else that i need to do?

**Comment by oizik** [comment]
Hi 

Did you ever resolve find a way around this?

**Comment by JackWormUK** [comment]
No.  I got fed up with this in the end and used Docker with this;

https://github.com/clean-docker/Magento2

Works perfectly
